core/drivers/crypto/se050/adaptors/utils/scp_config.c

```diff
@@ -438,8 +438,16 @@ bool se050_scp03_enabled(void)
 
 void se050_scp03_set_enable(enum se050_scp03_ksrc ksrc)
 {
+	static const char *const name[] = {
```

Comment:
right, didnt realize that, ok.

Yes it can not be set to a value that doesn't belong to the array. And these options are not going to change (there wont be any other key sources) so the assert is not really necessary but I'll protect anyway.